bigquery/integration_test.go

```diff
@@ -158,6 +158,7 @@ func initIntegrationTest() func() {
 		bqOpts := []option.ClientOption{option.WithTokenSource(ts)}
 		sOpts := []option.ClientOption{option.WithTokenSource(testutil.TokenSource(ctx, storage.ScopeFullControl))}
 		ptmOpts := []option.ClientOption{option.WithTokenSource(testutil.TokenSource(ctx, "https://www.googleapis.com/auth/cloud-platform"))}
+		connOpts := []option.ClientOption{option.WithTokenSource(testutil.TokenSource(ctx, connection.DefaultAuthScopes()...))}
```

Comment:
Maybe we just use the cloud platform scope directly here as well??  The default is bigquery + cloud platform, but cloud-platform already contains bigquery.

Reply:
I wonder if this is an error on the `connection.DefaultAuthScopes` method, as it returns both scopes. For me is easier to understand the code by using that method instead of putting a scope directly, but you're right, the cloud-platform scope would cover what is needed.